playbooks/preflight-check.yml

```diff
@@ -84,8 +86,30 @@
       when:
         - fs_check.stdout_lines
       with_items: "{{ fs_check.stdout_lines }}"
-  tags:
-    - leap-fs-check
+      tags:
+        - leap-fs-check
+
+    - name: Gather lxc filesystem used percentages
+      shell: |
+        df -h {{ lxc_container_directory | default('/var/lib/lxc') }} | tail -n +2 | sed s/%//g | awk '{if($5 > 50) print "Alert: "$5 "% full - "$0}'
+      register: lxc_fs_check
+      when:
+        - inventory_host in groups["os-infra_hosts"]
+        - enable_lxc_fs_check | default(true) | bool
+
+    # RLM-1371 detect if enough space for rootfs backup
+    - name: Check if lxc filesystem is greater than 50% full
+      debug:
+        msg: >-
+          The LXC filesystem(s) have been found to be greater than 50% full.
+          Details: {{ item.split() | join(' ') }}
+          If you have fully awareness of the potential risk having failure at rootfs backup stage, you can skip this check by passing the option 'enable_lxc_fs_check=false'.
+      failed_when: true
+      when:
+        - inventory_host in groups["os-infra_hosts"]
+        - lxc_fs_check.stdout_lines
+        - enable_lxc_fs_check | default(true) | bool
+      with_items: "{{ lxc_fs_check.stdout_lines }}"
```

Comment:
Got this when running it:

[DEPRECATION WARNING]: Skipping task due to undefined Error, in the future this will be a fatal error.: 'dict object' has no attribute 'stdout_lines'.
This feature will be removed in a future release. Deprecation warnings can be disabled by
setting deprecation_warnings=False in ansible.cfg.

Reply:
This one is very strange, and looks like it was triggered just because stdout content is including comma ':' mark. Since it's deprecation warning, do we still need to care about it?